qualify AtomMatrix in the rgpot RPC engine wrapper, so eOn 2.17.10 compiles against rgpot 3.0.0, where the type moved into rgpot::types
upstream fix: https://github.com/TheochemUI/eOn/commit/38c38c1bd3 (in eOn >= 3.0.0)
author: Rohit Goswami (SURF)
diff -ru eOn-2.17.10.orig/client/potentials/Rgpot/RGPotEngine.cpp eOn-2.17.10/client/potentials/Rgpot/RGPotEngine.cpp
--- eOn-2.17.10.orig/client/potentials/Rgpot/RGPotEngine.cpp	2026-09-21 14:06:28.986081537 +0200
+++ eOn-2.17.10/client/potentials/Rgpot/RGPotEngine.cpp	2026-09-21 14:06:29.085759802 +0200
@@ -18,6 +18,8 @@
 #include "rgpot/NWChemPot/NWChemPot.hpp"
 #include "rgpot/rpc/Potentials.capnp.h"
 
+using rgpot::types::AtomMatrix;
+
 namespace {
 
 std::string to_lower(std::string s) {
